### ISSUE TITLE
fuseops.FlushFileOp / fuseops.SyncFileOp errors

### ISSUE THREAD

**Issue by sniklaus** [GoogleCloudPlatform/gcsfuse#407]
I am getting fuseops.FlushFileOp / fuseops.SyncFileOp errors that prevent me from using Google Cloud Storage via gcsfuse.

```
Opening GCS connection...
Opening bucket...
Mounting file system...
File system has been successfully mounted.
fuse: 2020/07/06 14:25:16.572558 *fuseops.GetXattrOp error: function not implemented
2020/07/06 14:34:55.572371 Starting a garbage collection run.
2020/07/06 14:34:55.650214 Garbage collection succeeded after deleted 0 objects in 77.809742ms.
fuse: 2020/07/06 14:40:54.400979 *fuseops.SyncFileOp error: FileInode.Sync: SyncObject: Create: CreateObject: googleapi: got HTTP response code 503 with body: 
2020/07/06 14:44:55.572436 Starting a garbage collection run.
2020/07/06 14:44:55.783966 Garbage collection succeeded after deleted 0 objects in 211.504265ms.
fuse: 2020/07/06 14:53:54.731850 *fuseops.FlushFileOp error: FileInode.Sync: SyncObject: Create: CreateObject: googleapi: got HTTP response code 503 with body: 
2020/07/06 14:54:55.572573 Starting a garbage collection run.
2020/07/06 14:54:55.686077 Garbage collection succeeded after deleted 0 objects in 113.412116ms.
```

Any suggestions as to what may cause this? Could it be that using Ubuntu 20.04 LTS is causing this? Notice that these errors do not always occur, they seem to happen randomly. Thanks!

Please also see: #408, #410

**Comment by bjornleffler** [comment]
The code complains about Xattr opperations.

Are you using Extended attributes?
https://en.wikipedia.org/wiki/Extended_file_attributes

**Comment by sniklaus** [comment]
Thank you for the prompt response!

I understand that using Extended attributes may cause the fuseops.GetXattrOp error, but I was actually wondering about the other two errors. Could they be caused by Extended attributes as well? Please note that I am not familiar with Extended file attributes and am hence not sure whether I am using them or not. All I do is mounting a bucket with gcsfuse and then cp files over.

**Comment by sniklaus** [comment]
The more I run into this error the more it seems to relate to large files. I just tried to cp a file that is several gigabytes and cp failed with `failed to close '/mnt/gcs/thefile': Input/output error`. At the same time, gcsfuse shows a fuseops.FlushFileOp error.

**Comment by bjornleffler** [comment]
I'm looking at this. Thanks for reporting.

**Comment by sniklaus** [comment]
Seems to be fixed with #414, thanks!
